Dockerfile

```diff
@@ -50,7 +50,9 @@ COPY --from=cacher /tmp/$UNDERLAY_WS ./
 RUN . /opt/ros/$ROS_DISTRO/setup.sh && \
     apt-get update && rosdep install -q -y \
       --from-paths src \
-      --skip-keys slam_toolbox \
+      --skip-keys " \
+      slam_toolbox \
```

Comment:
You'll have to add this to the overlay build as well. Also handy to indent sub arguments for readability.

Reply:
I made the edits. Are there style guidelines or a linter that runs on these files that I can use in the future?

Reply:
There are a few best practices articles, but no really good linters for scripts.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices